test/TestProjects/RequestContextAllOptional-LowLevel/Generated/RequestContextAllOptionalClient.cs

```diff
@@ -172,13 +172,10 @@ public virtual Response RequestBodyResponseBody(RequestContent content, RequestC
         /// <summary> Delete. </summary>
         /// <param name="resourceName"> name. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="resourceName"/> is null. </exception>
 #pragma warning disable AZC0002
         public virtual async Task<Response> DeleteNoRequestBodyResponseBodyAsync(string resourceName, RequestContext context = null)
 #pragma warning restore AZC0002
         {
-            Argument.AssertNotNull(resourceName, nameof(resourceName));
```

Comment:
To make sure I understand: this was a query parameter, which is why we're removing validation?